x-pack/plugins/actions/server/actions_client.ts

```diff
@@ -283,6 +283,8 @@ export class ActionsClient {
       )
     );
 
+    await this.connectorTokenClient.deleteConnectorTokens({ connectorId: id });
```

Comment:
Depending on what we decide to do for the delete operation, we should do that same for update. If we're going to block the update operation if deleting connector tokens fails, we should move this to occur before the call to `unsecuredSavedObjectsClient.create` above.

Reply:
Done with this commit: https://github.com/elastic/kibana/pull/132061/commits/aef98fbf435fc21dc356c64a6f3d2922d9bc62a1.